src/entities/Discourse/types.ts

```diff
@@ -0,0 +1,11 @@
+export type DiscourseAttributes = {
+  address: string
+  forum_id: number
```

Comment:
Is this `forum_username` instead? 🤔

Reply:
`forum_username` could eventually change, forum_id is immutable